DEVELOPMENT.md

```diff
@@ -215,7 +215,7 @@ To run a single rekt test using the `e2e-debug.sh` script, follow these instruct
     **Example:**
 
     ```bash
-    ./hack/e2e-debug.sh TestMyRektScenario test/rekt/scenarios
+    ./hack/e2e-debug.sh TestPingSourceWithSinkRef /test/rekt
```

Comment:
The filepath needs to be relative, not absolute in order for this to work :)
```suggestion
    ./hack/e2e-debug.sh TestPingSourceWithSinkRef ./test/rekt
```

Reply:
Okay done.